docs/notes.md

```diff
@@ -878,5 +881,459 @@ With these observations, we can simplify the combined \\(m\\)-party statement ab
+involving \\({\mathbf{a}}\_{L}\\) and \\({\mathbf{a}}\_{O}\\) appear only on the left-hand side, terms
+involving \\({\mathbf{a}}\_{R}\\) appear only on the right-hand side, and
+non-secret terms (which the verifier can compute on its own) are
+factored out into a new term \\(\delta(y, z) \\).
+
+If we break apart the equation into individual terms, we can write it as:
+
+\\[
+\langle z \textbf{z}^Q,
+\textbf{c} + \textbf{W}\_V \cdot \textbf{v} \rangle =
+\langle \textbf{a}\_L \circ \textbf{a}\_R, \textbf{y}^n \rangle -
+\langle \textbf{a}\_O, \textbf{y}^n \rangle + 
+\langle z \textbf{z}^Q, 
+\textbf{W}\_L \cdot \textbf{a}\_L \rangle +
+\langle z \textbf{z}^Q, 
+\textbf{W}\_R \cdot \textbf{a}\_R \rangle +
+\langle z \textbf{z}^Q, 
+\textbf{W}\_O \cdot \textbf{a}\_O \rangle
+\\]
+
+Merge the statements containing \\(\textbf{a}\_O \\).
+
+\\[
+\langle z \textbf{z}^Q,
+\textbf{c} + \textbf{W}\_V \cdot \textbf{v} \rangle =
+\langle \textbf{a}\_L, 
+\textbf{y}^n \circ \textbf{a}\_R \rangle + 
+\langle \textbf{a}\_L,
+z \textbf{z}^Q \cdot \textbf{W}\_L \rangle +
+\langle \textbf{a}\_O, 
+-\textbf{y}^n + z \textbf{z}^Q \cdot \textbf{W}\_O \rangle +
+\langle \textbf{a}\_R, 
+z \textbf{z}^Q \cdot \textbf{W}\_R \rangle
+\\]
+
+Multiply the \\( \langle \textbf{a}\_R, 
+z \textbf{z}^Q \cdot \textbf{W}\_R \rangle \\) term by \\(\textbf{y}^n\\) one one side of the inner product and by \\(\textbf{y}^{-n}\\) on the other side:
+
+\\[
+\langle z \textbf{z}^Q,
+\textbf{c} + \textbf{W}\_V \cdot \textbf{v} \rangle =
+\langle \textbf{a}\_L, 
+\textbf{y}^n \circ \textbf{a}\_R \rangle + 
+\langle \textbf{a}\_L,
+z \textbf{z}^Q \cdot \textbf{W}\_L \rangle +
+\langle \textbf{a}\_O, 
+-\textbf{y}^n + z \textbf{z}^Q \cdot \textbf{W}\_O \rangle +
+\langle \textbf{y}^n \circ \textbf{a}\_R, 
+\textbf{y}^{-n} \circ (z \textbf{z}^Q \cdot \textbf{W}\_R) \rangle
+\\]
+
+Merge the statements containing \\(\textbf{y}^n \circ \textbf{a}\_R\\).
+
+\\[
+\langle z \textbf{z}^Q,
+\textbf{c} + \textbf{W}\_V \cdot \textbf{v} \rangle =
+\langle \textbf{a}\_L + \textbf{y}^{-n} \circ (z \textbf{z}^Q \cdot \textbf{W}\_R), 
+\textbf{y}^n \circ \textbf{a}\_R \rangle + 
+\langle \textbf{a}\_L,
+z \textbf{z}^Q \cdot \textbf{W}\_L \rangle +
+\langle \textbf{a}\_O, 
+-\textbf{y}^n + z \textbf{z}^Q \cdot \textbf{W}\_O \rangle
+\\]
+
+Add \\(\delta(y, z) = \langle \textbf{y}^{-n} \circ (z \textbf{z}^Q \cdot \textbf{W}\_R), z \textbf{z}^Q \cdot \textbf{W}\_L \rangle \\) to both sides. 
+
+\\[
+\langle z \textbf{z}^Q,
+\textbf{c} + \textbf{W}\_V \cdot \textbf{v} \rangle + \delta(y, z) =
+\langle \textbf{a}\_L + \textbf{y}^{-n} \circ (z \textbf{z}^Q \cdot \textbf{W}\_R), 
+\textbf{y}^n \circ \textbf{a}\_R \rangle + 
+\langle \textbf{a}\_L,
+z \textbf{z}^Q \cdot \textbf{W}\_L \rangle +
+\langle \textbf{a}\_O, 
+-\textbf{y}^n + z \textbf{z}^Q \cdot \textbf{W}\_O \rangle + 
+\langle \textbf{y}^{-n} \circ (z \textbf{z}^Q \cdot \textbf{W}\_R), z \textbf{z}^Q \cdot \textbf{W}\_L \rangle
+\\]
```

Comment:
The RHS of this equation is pretty long and it might be better to break it over multiple lines.